cms/templates/partials/hero.html

```diff
@@ -1,4 +1,22 @@
-{% load wagtailembeds_tags %}
+{% load wagtailembeds_tags wagtailimages_tags %}
+
+{% block extrahead %}
+  {% if page.background_image %}
+  <style>
```

Comment:
is there a reason that we can not put this css in .scss file?

Reply:
The `{{ background_image.url }}` needs to be done within a Django template.